SUMMARY.md

```diff
@@ -19,6 +19,7 @@
 
 * [Command handling](implementing-domain-logic/command-handling/command-handling.md)
     * [Aggregate](implementing-domain-logic/command-handling/aggregate.md)
+    * [Aggregate Polymorphism](implementing-domain-logic/command-handling/aggregate-polymorphism.md)
```

Comment:
I'd argue users would firstly start building multi-entity aggregates, after which aggregate polymorphism might become interesting to them.

Reply:
Don't have much of an opinion here, so, I'll move it one row below :P

Reply:
Great success!